pkg/openstack/placement.go

```diff
@@ -0,0 +1,54 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openstack-k8s-operators/lib-common/modules/common/helper"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	corev1beta1 "github.com/openstack-k8s-operators/openstack-operator/apis/core/v1beta1"
+	placementv1 "github.com/openstack-k8s-operators/placement-operator/api/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// ReconcilePlacementAPI -
+func ReconcilePlacementAPI(ctx context.Context, instance *corev1beta1.OpenStackControlPlane, helper *helper.Helper) (ctrl.Result, error) {
+	placementAPI := &placementv1.PlacementAPI{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "placement",
+			Namespace: instance.Namespace,
+		},
+	}
+
+	helper.GetLogger().Info("Reconciling placementAPI", "placementAPI.Namespace", instance.Namespace, "placementAPI.Name", "keystone")
```

Comment:
Based on the discussion in https://coreos.slack.com/archives/CQXJFGMK6/p1663595278584359 we should use the logging wrappers from https://github.com/openstack-k8s-operators/lib-common/blob/master/modules/common/util/log.go But probably that is for a separate patch to replace all the helper.GetLogger() calls in the openstack-operator.

Reply:
agreed